base/commands/sql/sql_test.go

```diff
@@ -0,0 +1,113 @@
+package sql_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	_ "github.com/hazelcast/hazelcast-commandline-client/base/commands"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/it"
+)
+
+func TestSQL(t *testing.T) {
+	testCases := []struct {
+		name string
+		f    func(t *testing.T)
+	}{
+		{name: "SQL_Interactive", f: sql_InteractiveTest},
+		{name: "SQL_NonInteractive", f: sql_NonInteractiveTest},
+		{name: "SQLOutput_NonInteractive", f: sqlOutput_NonInteractiveTest},
```

Comment:
FYI tests are not sorted.

Reply:
Sorted at: https://github.com/hazelcast/hazelcast-commandline-client/pull/198/commits/32be06c01b7cba3b8b9161145a6c37f562d3e9fc